tests/rptest/clients/rpk.py

```diff
@@ -643,6 +651,43 @@ def produce(self,
         assert m, f"Reported offset not found in: {out}"
         return int(m.group(1))
 
+    def analyze_topic(self, topic: str,
+                      time_range: str) -> Iterator[RpkAnalyzedTopic]:
+        cmd = ['analyze', topic, '-t', time_range, '--print-topics']
```

Comment:
FWIW, now that you added support to the `--format` flag, it might be easier to use that + json.parse in these ducktape tests.

Reply:
I'm avoiding it as the json output is the raw un-aggregated data rather than the aggregated averages/sums that are outputted in the tables. So by using the tables instead I avoid having to re-implement the aggregation in python and can further validate that the aggregation that occurs in RPK is as expected.